### DISCUSSION TITLE
Does this package support near blockchain?

### DISCUSSION THREAD

**Discussion by Coding-Algorithm** [heroims/react-native-web3-wallet#5] [Q&A]
Hi.

I will like to thank the contributor for this life-saving tools they put out here..

I will like to ask if this repo supports the near the blockchain. I am working on a project that needs to transact using the near coin.

Thanks, awaiting your reply.

**Comment by heroims** [comment]
No，near is not ethereum network, you must use [Near's Official Tools](https://github.com/near/near-api-js)
